src/Compute/Compute/Generated/GalleryImage/GalleryImageCreateOrUpdateMethod.cs

```diff
@@ -185,7 +188,19 @@ public override void ExecuteCmdlet()
 
                     if (this.IsParameterBound(c => c.Feature))
                     {
-                        galleryImage.Features = this.Feature;
+                        galleryImage.Features = new List<GalleryImageFeature>();
+                        for (int i = 0; i < this.Feature.Length; i++)
+                        {
+                            galleryImage.Features.Add(this.Feature[i]);
+                        }
+                    }
+                    if (SecurityTypeNotFoundInFeatures(galleryImage.Features) && galleryImage.HyperVGeneration == "V2")// if SecurityType is not in Features, default to TrustedLaunch
```

Comment:
Suggestion: Can you collapse the separate function into 1 single line check?

if (galleryImage.Features?.All(f => f.Name.ToLower() != "securitytype") == true && galleryImage.HyperVGeneration == "V2")

Reply:
thank you for the cool notation. Yes, one line code is better than a function. 

I had to add one more conditional for when Feature is not passed in at all, but this notation works well